registration_form.py

```diff
@@ -0,0 +1,262 @@
+import logging
+import random
+from time import sleep
+
+from selenium.webdriver.chrome.webdriver import WebDriver
+from selenium.webdriver.common.by import By
+
+
+class TestStartPage:
```

Comment:
As registration form is part of the start page, lest store these tests in `test_start_page.py`